### PR TITLE
Fix the CUDNN_BN_MIN_EPSILON difference issue between cudnn7.3 and cudnn7.6

### PR DESCRIPTION
**Pull request by yufenglee**
In cudnn 7.6, CUDNN_BN_MIN_EPSILON is 0.0, however, it is 1e-5 in cudnn 7.3. It leads to the difference in LayerNorm with its original subgraph

cudnn 7.6
#define CUDNN_BN_MIN_EPSILON 0.0 /* Minimum epsilon allowed to be used in the Batch Normalization formula */
 
cudnn 7.3:
#define CUDNN_BN_MIN_EPSILON 1e-5 /* Minimum epsilon allowed to be used in the Batch Normalization formula */

### CODE REVIEW

**Comment by tianleiwu** [comment]
Have you tested epsilon < e-5 with cudnn 7.3? I saw comments related to this in [TF issue](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn_impl.py#L1481):
> Set a minimum epsilon to 1.001e-5, which is a requirement by CUDNN to prevent exception (see cudnn.h).

I suggest not change the code. Instead build and publish binary with cudnn 7.6.

**Comment by yufenglee** [comment]
> Have you tested epsilon < e-5 with cudnn 7.3? I saw comments related to this in TF issue:
> 
> Set a minimum epsilon to 1.001e-5, which is a requirement by CUDNN to prevent exception (see cudnn.h).
> 
> I suggest not change the code. Instead build and publish binary with cudnn 7.6.

It should be good here because LayerNorm is not using cudnn for the computaton
